implementation.js

```diff
@@ -2,14 +2,18 @@
 
 // modified from https://github.com/es-shims/es6-shim
 var keys = require('object-keys');
-var bind = require('function-bind');
 var canBeObject = function (obj) {
 	return typeof obj !== 'undefined' && obj !== null;
 };
+var bind = function (fn, thisArg) {
+	return function () {
+		return fn.apply(thisArg, arguments);
```

Comment:
either way, this won't actually work - `delete Function.prototype.apply` will break this "bind" implementation. I think `eval` is actually *always* required in an ES3 environment.